packages/backend/src/models/flow.ts

```diff
@@ -48,6 +49,14 @@ class Flow extends Base {
           rejectIfOverMaxQps: {
             type: 'boolean',
           },
+          errorConfig: {
+            type: 'object',
+            properties: {
+              notificationFrequency: {
+                type: 'string',
+              },
+            },
+          },
```

Comment:
just wanted to check if this is for validation? cus before i added this, everything seems to work as per normal